p8e-api/src/main/kotlin/io/provenance/engine/grpc/observers/EnvelopeEventObserver.kt

```diff
@@ -168,6 +169,8 @@ class EnvelopeEventObserver(
     private fun connect(value: EnvelopeEvent): QueueingStreamObserverSender<EnvelopeEvent> {
         val publicKey = if(value.publicKey.hasEncryptionPublicKey()) value.publicKey.encryptionPublicKey.toPublicKey() else value.publicKey.signingPublicKey.toPublicKey()
 
+        logger().info("Connecting [publicKey = ${publicKey.toHex()}, classname = ${value.classname}, action = ${value.action}, ip = ${clientIp()}]")
```

Comment:
I think it makes sense to have this at the debug level, that would require us finding a way to control the logging level outside of this repo and with p8e-deployment or the figure-deployment-controller directly.